decentraland/unity-renderer
# Feat: new notification system

## What does this PR change?

Fix #2399

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat%2Fnotifications&ENABLE_NOTIFICATION_CHAT=&realm=dg&position=100%2C100&island=I9xvf
2. By pressing K you can generate new notitifaction
3. Verify that clicking on notifications open the correct chat
4. check the fadeout after 8 second of the notifications
5. Check that the panel doesn't fade if you mouse hover

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
